_includes/v20.2/prod-deployment/insecure-start-nodes.md

```diff
@@ -24,6 +24,35 @@ After completing these steps, nodes will not yet be live. They will complete the
     | tar  xvz
     ~~~
 
+echo 'Installing CockroachDB v20.2 and supporting spatial libraries...'
+
+wget -qO- https://binaries.cockroachdb.com/cockroach-v20.2.5.linux-amd64.tgz | tar  xvz
```

Comment:
It looks like this unrendered section might not be necessary / can be deleted?  Since it appears to duplicate what is below.